backend/api/Database/Context/FlotillaDbContext.cs

```diff
@@ -69,11 +69,13 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             poseBuilder.OwnsOne(pose => pose.Position);
             poseBuilder.OwnsOne(pose => pose.Orientation);
         });
-        modelBuilder.Entity<Area>().HasOne(a => a.Deck).WithMany();
+        modelBuilder.Entity<Area>().HasOne(a => a.Deck).WithOne(d => d.DefaultLocalizationArea);
         modelBuilder.Entity<Area>().HasOne(a => a.Installation).WithMany();
         modelBuilder.Entity<Area>().HasOne(a => a.Plant).WithMany();
         modelBuilder.Entity<Deck>().HasOne(d => d.Plant).WithMany();
+
         modelBuilder.Entity<Deck>().HasOne(d => d.Installation).WithMany();
+        modelBuilder.Entity<Deck>().HasOne(d => d.DefaultLocalizationArea).WithOne(a => a.Deck);
```

Comment:
Does `d` here mean `deck` while `a` points to an `DefaultLocalizationArea` and not an `Area`?

Reply:
Or I guess `DefaultLocalizationArea` is actually an `Area` object?